packages/core/src/connectors/aliyun-sms/index.ts

```diff
@@ -0,0 +1,107 @@
+import { PasscodeType } from '@logto/schemas';
+import { z } from 'zod';
+
+import {
+  ConnectorConfigError,
+  ConnectorError,
+  ConnectorMetadata,
+  ConnectorType,
+  SmsSendMessageFunction,
+  ValidateConfig,
+} from '../types';
+import { getConnectorConfig } from '../utilities';
+import { sendSms } from './single-send-text';
+
+export const metadata: ConnectorMetadata = {
+  id: 'aliyun-sms',
+  type: ConnectorType.SMS,
+  name: {
+    en: 'Aliyun Short Message Service',
+    zh_CN: '阿里云短信服务',
+  },
+  logo: './logo.png',
+  description: {
+    en: 'Short Message Service (SMS) has a batch sending feature and various API operations to send one-time password (OTP) messages, notification messages, and promotional messages to customers.',
+    zh_CN:
+      '短信服务（Short Message Service）是指通过调用短信发送API，将指定短信内容发送给指定手机用户。',
+  },
```

Comment:
Can we put these phrases in the i18n module?

Reply:
Connectors will be separated as a monorepo to support the functionality of logto/core, so it's better to keep it in place?

Reply:
OK, I'm fine with it.